components/Header/SliderHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import { Swiper, SwiperSlide } from 'swiper/react';
+import ShoesImg from '../../images/shoesImg.jpg';
+import BikeImg from '../../images/bikeImg.jpg';
+import TennisImg from '../../images/tennisImg.jpg';
+import { Autoplay } from 'swiper';
+import Image from 'next/image';
+
+const imgages = [
+	{ src: ShoesImg, name: 'ShoesImg', alt: 'Basketball shoes' },
+	{ src: BikeImg, name: 'BikeImg', alt: 'Stationary bikes' },
+	{ src: TennisImg, name: 'TennisImg', alt: 'Tennis racket' },
+];
+
+export default function SliderHeader() {
+	return (
+		<>
+			<Swiper
+				autoplay={{
+					delay: 4000,
+					disableOnInteraction: false,
+				}}
+				loop={true}
+				modules={[Autoplay]}
+				className='mySwiper'>
+				{imgages.map((photo) => {
```

Comment:
typo imgages -> images
<img width="452" alt="image" src="https://github.com/firefighterTom/project-eshop/assets/71249791/9ae5bd77-ac2a-4f33-9dfc-0d41f3faae35">
Download the extension